clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -728,20 +725,14 @@ synchronized public ListConsumerGroupOffsetsResult listConsumerGroupOffsets(Map<
         String group = groupSpecs.keySet().iterator().next();
         Collection<TopicPartition> topicPartitions = groupSpecs.get(group).topicPartitions();
         final KafkaFutureImpl<Map<TopicPartition, OffsetAndMetadata>> future = new KafkaFutureImpl<>();
-
-        if (listConsumerGroupOffsetsException != null) {
-            future.completeExceptionally(listConsumerGroupOffsetsException);
+        if (topicPartitions.isEmpty()) {
```

Comment:
Maybe we can move `topicPartitions.isEmpty()` into the `filter`.
```java
        future.complete(committedOffsets.entrySet().stream()
            .filter(entry -> topicPartitions.isEmpty() || topicPartitions.contains(entry.getKey()))
            .collect(Collectors.toMap(Map.Entry::getKey, entry -> new OffsetAndMetadata(entry.getValue()))));
```

WDYT?

Reply:
Done, thanks for the suggestion!